app/models/ApplicationRequest.scala

```diff
@@ -56,67 +67,137 @@ case class EORIDetails(
   postcode: String,
   country: String
 )
+object EORIDetails {
+  implicit val format: OFormat[EORIDetails] = Json.format[EORIDetails]
+}
 
 case class ContactDetails(
   name: String,
   email: String,
   phone: Option[String]
 )
+object ContactDetails {
+  implicit val format: OFormat[ContactDetails] = Json.format[ContactDetails]
+}
 
-/** determined via `_type` field, e.g. "_type" = "MethodOne" { "_type": "MethodOne",
-  * "saleBetweenRelatedParties": "blah blah", ...
-  */
 sealed trait RequestedMethod
 
 case class MethodOne(
   saleBetweenRelatedParties: Option[String],
   goodsRestrictions: Option[String],
   saleConditions: Option[String]
```

Comment:
I could be wrong but I think these fields are Yes/No questions. Should we be using an enum instead of a String?